src/Model/Table/RequestsTable.php

```diff
@@ -79,22 +80,23 @@ public function gc()
         if (time() % 100 !== 0) {
             return;
         }
-        $purge = $this->find()
+        $noPurge = $this->find()
             ->select(['id'])
             ->hydrate(false)
-            ->where(['requested_at <=' => new \DateTime('-2 weeks')])
+            ->order(['requested_at' => 'desc'])
+            ->limit(Configure::read('DebugKit.requestCount') ?: 20)
```

Comment:
Does it hurt (in size or performance) to keep a little bit more, e.g. 50 or 100, around?

With a little more request history, it will enable people to analyze the logs without having to configure a higher count and then go and build up the history first.